packages/fluent-theme/src/fluent/styles/BasePicker.styles.ts

```diff
@@ -1,13 +1,18 @@
 import { IBasePickerStyleProps, IBasePickerStyles } from 'office-ui-fabric-react/lib/Pickers';
-import { fluentBorderRadius } from './styleConstants';
 
 export const BasePickerStyles = (props: IBasePickerStyleProps): Partial<IBasePickerStyles> => {
+  const { theme } = props;
+  if (!theme) {
+    throw new Error('Theme is undefined or null.');
```

Comment:
shouldn't this never happen? isn't a base part of fabric that we guarantee the theme?

Reply:
Some of the components don't follow this pattern in their style props to make the `theme` required and this happens. I believe that is something that needs to be cleaned up as well to have a consistent API across all components.

Reply:
Whether or not theming is required via props typing doesn't affect whether it's actually available. This is actually determined by the use of `styled` helper which will automatically inject `themes` prop, even if the component has made `theme` prop optional. I've modified this approach in `Foundation` by making theme prop required as part of an inherited interface.

For now, no other components are actually checking for theme's existence in styles functions, so I think it's consistent to assume it's available in styles functions.

Reply:
(The assumption here is that 'some of the components' are all components using `styled`. Components that use `customizable` or other methods cannot technically have styles functions called.)

Reply:
@JasonGore if I remove the theme check on this components I get this lint message:
![screen shot 2019-02-25 at 10 50 11 am](https://user-images.githubusercontent.com/29514833/53365415-09db7b80-38f6-11e9-843d-636c86033ef1.png)
Also, there are a couple of components in OUFR that are still doing the same thing. And I know that in Foundation this is fixed 👍  Thanks for the consisitency in the API that it will bring to our repo.

Reply:
Yes, this is because of the typing on the component making it optional. Other styles functions do this:
```tsx
const { effects } = theme!;
```

Which of course isn't ideal either. In either case an exception would occur. Maybe your approach is best for communicating the actual issue.